### PR TITLE
Add index and reference documentation

### PR DESCRIPTION
**Pull request by megian**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Update the ./CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 
-- …
+## [0.0.1] - 2020-09-29
```
Comment (simu) [inline CHANGELOG.md:10]: We've generally tagged our releases as `v0.0.1`

**Review by simu — APPROVED** [review]
LGTM

**Review by Kidswiss — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (megian) [reply CHANGELOG.md]: Removed the version, as we can argue that this is not yet a release.

**Pull request merged**